OpenRA.Mods.Common/Traits/Armament.cs

```diff
@@ -63,6 +63,8 @@ public class ArmamentInfo : ConditionalTraitInfo, Requires<AttackBaseInfo>
 		[Desc("Use multiple muzzle images if non-zero")]
 		public readonly int MuzzleSplitFacings = 0;
 
+		public readonly bool ShootAtClosestTargetablePosition = false;
```

Comment:
Will that hit this PR? If not, then I'd be more than okay with going that `TargetAcquisition` list for now and expand it with a new entry when that customtargetingpositions logic arrives. Bleeding this through `ProjectileArgs` is definitely meh - so does the fact we really need this but I can live with that latter. >_>